src/sql/src/session/vars/definitions.rs

```diff
@@ -2116,6 +2116,12 @@ feature_flags!(
         default: true,
         enable_for_item_parsing: false,
     },
+    {
+        name: enable_reduce_unnest_list_fusion,
+        desc: "Enables fusing `Reduce` with `FlatMap UnnestList` for better window function performance",
+        default: false,
```

Comment:
Why isn't this enabled by default? It's fine with me not to enable it, but I'd like to understand the reasoning!

Reply:
Originally, the plan was to
1. Roll this PR out as a patch release to _all_ customers this week, in which case it seemed less risky to enable it only for GM this week.
2. Enable it for all customers in the next release window. I was planning to do this in a follow-up PR this Thursday, where I'd set this default to `true`.

However, then Nikhil decided (completely understandably) that we should roll this PR out in a special patch release to _only_ GM (tomorrow, right after the normal release goes out). So, now I think I'll modify this PR to make it enabled by default.

(Generally, LD can of course override whatever default we set here, but I think it's good if the default value here is consistent with the LD setting of the majority of customers, so that we run the entirety of CI with the common setting.)